posthog/temporal/data_imports/pipelines/pipeline.py

```diff
@@ -120,7 +120,10 @@ def _run(self) -> dict[str, int]:
                         if e.exception.args[0] != "Generic error: No data source supplied to write command.":
                             raise
 
-                row_counts = pipeline.last_trace.last_normalize_info.row_counts
+                if pipeline.last_trace.last_normalize_info is not None:
+                    row_counts = pipeline.last_trace.last_normalize_info.row_counts
+                else:
+                    row_counts = {}
                 # Remove any DLT tables from the counts
                 filtered_rows = dict(filter(lambda pair: not pair[0].startswith("_dlt"), row_counts.items()))
```

Comment:
nit: Since we are only accessing the first item of the tuple yielded by items (the key), we could just use `row_counts.keys()` instead of `row_counts.items()`.

Reply:
Out of scope though, I just read this line as I was reviewing the above. Can be tackled in a later PR.